translations/town.tex/pl.po

```diff
@@ -80,7 +80,21 @@ msgid ""
 "Some Building Tiles are double-sided, and may later be upgraded and flipped to represent two different buildings at the same time. Such upgrades must be \\textbf{Built in order}.\\par\n"
 "\n"
 msgstr ""
-
+"\\hypertarget{Miasto}{Każda}\\index{Miasto} Frakcja ma własne Miasto, które znajduje się na środku swojego Kafelka Początkowego.\n"
+"Miasto jest twoją najważniejszą Lokacją - wiele Scenariuszy może się \\hyperlink{End}{zakończyć}, jeśli \\hyperlink{Categories}{oflaguje} je Bohater wroga.\\par\n"
+"Stan twojego Miasta i Frakcji są ukazane na Planszy Miasta.\n"
+"Pokazane są na niej wybudowane Budynki, Koszt budowy dostępnych Budynków, przychód zasobów i stan Żetonów Akcji Miasta.\\par\n"
+"Każda Frakcja może wybudować następujące Budynki\\index{Buildings} w swoim Mieście:\n"
+"\\begin{itemize}\n"
+"  \\item \\textbf{Ratusz} – W rundzie Zasobów, zyskujesz przychód albo korzystasz z unikatowej zdolności swojej Frakcji.\n"
+"  \\item \\textbf{Cytadela} – Pozwala Wzmacniać jednostki za pomocą Żetonu Populacji.\n"
+"Ponadto \\hyperlink{Mury}{chronią twoje Miasto} podczas oblężenia.\n"
+"  \\item \\textbf{Siedliska Jednostek} – Pozwalają Rekrutować Jednostki.\n"
+"Jednostki należy odblokować budując Siedliska w kolejności:\\includesvg[height=10px]{\\svgs/bronze.svg}\\includesvg[height=10px]{\\svgs/silver.svg}\\includesvg[height=10px]{\\svgs/Golden.svg}\n"
```

Comment:
```suggestion
"Siedliska umożliwiają rekrutowanie jednostek, wznosić je można tylko w kolejności:\\includesvg[height=10px]{\\svgs/bronze.svg}\\includesvg[height=10px]{\\svgs/silver.svg}\\includesvg[height=10px]{\\svgs/Golden.svg}\n"
```

Zdanie przeinaczone, ale mam wrażenie, że lepiej oddaje kontekst. Jakby nie patrzeć nie ma konieczności budowania siedlisk, jest za to wymuszenie żeby zachować kolejność